makeflow/src/makeflow.c

```diff
@@ -1425,6 +1519,21 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	// check to ensure that all jobs can be ran with resources given
+	if(clean_mode == MAKEFLOW_CLEAN_ALL)	
+	{	
+		int tooBig;
```

Comment:
tooBig should be declared outside of this if statement. Also, loop over all of the nodes regardless of it we are in clean mode. Then check tooBig and clean mode before exiting.